scripts/partition_manager.py

```diff
@@ -935,19 +935,19 @@ def main():
 def expect_addr_size(td, name, expected_address, expected_size):
     if expected_size:
         assert td[name]['size'] == expected_size, \
-            "Size of {} was {}, expected {}.\ntd:{}".format(name, td[name]['size'], expected_size, pformat(td))
+            'Size of {} was {}, expected {}.\ntd:{}'.format(name, td[name]['size'], expected_size, pformat(td))
```

Comment:
Should we get rid of all `.format(` while we are at it? Feel free to say no, since this is already a huge improvement :)

Reply:
I understand that it still might be useful in when the same variable is referenced several times, but I see there are places still where this is not the case.

Reply:
I guess it can help with readability in some cases as well. Anyways, I'd say feel free to judge for yourself which is better.

Reply:
The way I look at it is that `.format(` calls are nicer when you're inserting from dicts. F-strings with embedded key-access start to mess with the apostrophe/quote usage that can be hard to notice unless your editor helps you out visually.

F.eks
`f'Something something {adict["with a key"]}'` <-- must be conscious about the apostrophe/quotes when accessing the key, otherwise syntax errors.

`'Something something {}'.format(adict['with a key'])`  <-- no thinking about quote/apostrophe usage across the line.

I think it mostly comes down to personal preference. I tend to be a bit of a purist in Python with all apostrophes all the time, and I was happy to see that be the consistent majority with these scripts already. I typically only ever use quotations with docstrings or if the string has a contraction in it like `s = "Don't do that."`. Even then, if I'm feeling saucy I might just `Don\'t`.

Totally fine if we want to use f-strings all the time and do away with `.format(` whenever possible though!

Reply:
Good point about the dict keys.